### ISSUE TITLE
Add option for different .takeoff folder

### ISSUE THREAD

**Issue by Narigo** [entwicklerstube/takeoff#14]
Right now it is mandatory to use a `.takeoff` directory where the stations reside. It would be nice to be able to put it somewhere else (for example `config/takeoff`, if I already have a config directory for a lot of other things)

**Comment by michaelzoidl** [comment]
Good idea, will write it into the next-features-in-major-release list! :)
